packages/rocket-webhook-core/src/webhook-dispatcher.ts

```diff
@@ -25,6 +26,15 @@ export async function dispatch(
     console.log('Webhook: request received')
     const requestId = UUID.generate()
     const webhookParamsEvent = getWebhookParamsEvent(params, request)
+
+    // Default allowedMethods to [POST] if not defined
+    const allowedMethods = webhookParamsEvent.allowedMethods ?? AllowedHttpMethod.POST
```

Comment:
Shouldn't it be an array with the POST method?
```suggestion
    const allowedMethods = webhookParamsEvent.allowedMethods ?? [AllowedHttpMethod.POST]
```

Reply:
Fixed in commit a7212c64a7e659972a749aa9ada924be3d1d85b5.